### PR TITLE
[DF] Support rdfslot_ and rdfentry_ as well as avoid duplication of routines to check implicit columns

### PR DESCRIPTION
**Pull request by dpiparo**

### CODE REVIEW

**Comment by bluehood** [comment]
Hi, since we are touching this, is the feature clearly indicated in the docs?

File: tree/dataframe/src/RDFInterfaceUtils.cxx [hunk 1]
```diff
@@ -349,7 +349,7 @@ ColumnNames_t FindUnknownColumns(const ColumnNames_t &requiredCols, const Column
 
 bool IsInternalColumn(std::string_view colName)
 {
-   return 0 == colName.find("tdf") && '_' == colName.back();
+   return (0 == colName.find("tdf") || 0 == colName.find("rdf")) && '_' == colName.back();
```
[the remark this answers is not in the document]
Comment (dpiparo) [reply tree/dataframe/src/RDFInterfaceUtils.cxx]: uhm, is this noticeable for a <10 char string?

**Comment by dpiparo** [comment]
For the docs, good idea. Never enough.

**Comment by dpiparo** [comment]
merged after addressing the documentation comment, too.

**Pull request merged**